albumentations/augmentations/bbox_utils.py

```diff
@@ -147,4 +159,60 @@ def convert_bboxes_from_albumentations(shape, bboxes, target_format):
         bboxes (list): List of bounding box with coordinates in the format used by albumentations
         target_format (str): required format of the output bounding box. Should be 'coco' or 'pascal_voc'.
     """
-    return [convert_bbox_from_albumentations(shape, bbox, target_format) for bbox in bboxes]
+    return [convert_bbox_from_albumentations(bbox, target_format, rows, cols, check_validity) for bbox in bboxes]
+
+
+def check_bbox(bbox):
+    """Check if bbox boundaries are in range 0, 1 and minimums are lesser then maximums"""
+    for name, value in zip(['x_min', 'y_min', 'x_max', 'y_max'], bbox[:4]):
+        if not 0 <= value <= 1:
+            raise ValueError(
+                'Expected {name} for bbox {bbox} '
+                'to be in the range [0.0, 1.0], got {value}.'.format(
+                    bbox=bbox,
+                    name=name,
+                    value=value,
+                )
+            )
+    x_min, y_min, x_max, y_max = bbox[:4]
+    if x_max <= x_min:
+        raise ValueError('x_max is less than or equal to x_min for bbox {bbox}.'.format(
+            bbox=bbox,
+        ))
+    if y_max <= y_min:
+        raise ValueError('y_max is less than or equal to y_min for bbox {bbox}.'.format(
+            bbox=bbox,
+        ))
+
+
+def check_bboxes(bboxes):
+    """Check if bboxes boundaries are in range 0, 1 and minimums are lesser then maximums"""
+    for bbox in bboxes:
+        check_bbox(bbox)
+
+
+def filter_bboxes(bboxes, rows, cols, min_area=0., min_visibility=0.):
+    """Remove bounding boxes that either lie outside of the visible area by more then min_visibility
+    or whose area in pixels is under the threshold set by `min_area`. Also it crops boxes to final image size.
+
+    Args:
+        min_area (float): minimum area of a bounding box. All bounding boxes whose visible area in pixels
+            is less than this value will be removed. Default: 0.0.
+        min_visibility (float): minimum fraction of area for a bounding box to remain this box in list. Default: 0.0.
+        rows (int): Image rows.
+        rows (int): Image cols.
+    """
+    resulting_boxes = []
+    for bbox in bboxes:
+        if min_visibility:
+            transformed_box_area = calculate_bbox_area(bbox, rows, cols)
+            bbox[:4] = np.clip(bbox[:4], 0, 1.)
+            clipped_box_area = calculate_bbox_area(bbox, rows, cols)
+            if clipped_box_area / transformed_box_area < min_visibility:
```

Comment:
Not sure, but maybe we need to check that `transformed_box_area` is not 0 to avoid ZeroDivisionError.